Forklift/PlanPart.cs

```diff
@@ -81,7 +81,7 @@ public void Update(PlanPart parent, UpdateContext context)
             Table = context.Metabase.Table(TableName);
             Columns = Table.Columns;
             Ignore(Columns.Where(x => x.Insertable() ==false).Select(x => x.Name).ToArray());
-            Ignore(Table.PrimaryKey);
+            //Ignore(Table.PrimaryKey);
```

Comment:
This change is probably a bit controversial.

I exclusively use guid ids and wanted to keep the ids in there, obviously this doesn't work so well for identity columns.

Any ideas on a better way to deal with this for both cases?